packages/core/components/EditMetadata/MetadataDetails.tsx

```diff
@@ -41,20 +52,95 @@ export default function EditMetadataDetailsList(props: DetailsListProps) {
         }
         return <></>;
     };
+    const annotationFormatter = annotationFormatterFactory(
+        props.fieldType || AnnotationType.STRING
+    );
 
     function renderItemColumn(
         item: ValueCountItem,
         _: number | undefined,
         column: IColumn | undefined
     ) {
         const fieldContent = item[column?.fieldName as keyof ValueCountItem] as string;
-        if (!fieldContent) return "[No value] (blank)";
-        if (column?.fieldName === "fileCount") {
+        if (column?.fieldName === "value") {
+            if (!fieldContent) return "[No value] (blank)";
+            else return annotationFormatter.displayValue(fieldContent);
+        } else if (column?.fieldName === "fileCount") {
```

Comment:
so fileCount will always have a value?

Reply:
it should. I'll add a check for clarity though